package.json

```diff
@@ -43,9 +59,10 @@
     "jest": "^27.2.0",
     "rimraf": "^3.0.2",
     "ts-jest": "^27.0.5",
-    "typescript": "^4.3.5"
+    "typescript": "^5.6.3"
   },
-  "dependencies": {
+  "dependencies": {    
+    "@types/node": "^22.8.7",
```

Comment:
This will most probably break this plugin when installed by a frontend library. What do you think?

Reply:
This is tried and developed together with the React-plugin. so should work. 
For our own internal stuff we will need to update things quite a lot anyways. Which is

Reply:
So @types packages should probably not ship as regular dependency but install as a devDependency. Why do we need to export this?